admin-dev/themes/default/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: [
+    'cldrjs/dist/cldr.js',
```

Comment:
In fact, all my problems are due to the import on the single entry point.

As we call babel and the es-2015 plugin, all the modules are not imported the same way as in the single entry point. This method fix everything.